sklearn/tests/test_public_functions.py

```diff
@@ -124,6 +124,7 @@ def test_function_param_validation(func_module):
 
 PARAM_VALIDATION_CLASS_WRAPPER_LIST = [
     ("sklearn.decomposition.non_negative_factorization", "sklearn.decomposition.NMF"),
+    ("sklearn.cluster.affinity_propagation", "sklearn.cluster.AffinityPropagation"),
```

Comment:
I made this change here.
We modified the code such that we check all parameters, even the ones that are not provided. To be able to do so we need to provide the relation between the function and the class.